### PR TITLE
add getMe!

### PR DESCRIPTION
**Pull request by thoriqazzikra**

### CODE REVIEW

**Comment by harieyadi** [comment]
Error

**Comment by thoriqazzikra** [comment]
> Error

Errornya dmna?

**Comment by harieyadi** [comment]
Di author

**Comment by thoriqazzikra** [comment]
> Di author

coba di ss atau di copas

**Comment by Fucksociety666** [comment]
> > Di author
> 
> coba di ss atau di copas

Screenshot pada 2020-12-07 21-50-17

**Comment by TheSploit** [comment]
> > > Di author
> > 
> > 
> > coba di ss atau di copas
> 
> Screenshot pada 2020-12-07 21-50-17

di command profile, sesuaikan dlu tu yg isAdminsny dgn yg di Aruga, atau engga tambahin ajh jadi gini :
const isAdmin = adminNumber.includes(pengirim)(yg pengirim bisa juga sih diganti jadi sender.id

**Comment by thoriqazzikra** [comment]
> > > > Di author
> > > 
> > > 
> > > coba di ss atau di copas
> > 
> > 
> > Screenshot pada 2020-12-07 21-50-17
> 
> di command profile, sesuaikan dlu tu yg isAdminsny dgn yg di Aruga, atau engga tambahin ajh jadi gini :
> const isAdmin = adminNumber.includes(pengirim)(yg pengirim bisa juga sih diganti jadi sender.id

Udh ditambahin ngab, cmn g kedetect, masih 'isAdmin' is not undefined
Klo gua sih delete aja yg isAdminnya

**Comment by thoriqazzikra** [comment]
> > > Di author
> > 
> > 
> > coba di ss atau di copas
> 
> ```
>             await aruga.sendFileFromUrl(from, pfp, 'pfp.jpg', `*User Profile* ✨️ \n\n➸ *Username: ${namae}*\n\n➸ *User Info: ${status}*\n\n➸ *Admin Group: ${adm}*\n\n`)
>          } else if (quotedMsg) {
>          var qmid = quotedMsgObj.sender.id
>          var pic = await aruga.getProfilePicFromServer(qmid)
>          var namae = quotedMsgObj.sender.name
>          var sts = await aruga.getStatus(qmid)
>          var adm = isGroupAdmins
> ```
> 
> 159 var donate = isAdmin
> const { status } = sts
> if (pic == undefined) {
> var pfp = errorurl
> } else {
> var pfp = pic
> }
> await aruga.sendFileFromUrl(from, pfp, 'pfp.jpg', `*User Profile* ✨️ \n\n➸ *Username: ${namae}*\n\n➸ *User Info: ${status}*\n\n➸ *Admin Group: ${adm}*\n\n`)
> }
> }
> break
> Screenshot pada 2020-12-07 21-50-17

Nanti gua update lgi, lupa masukin const errorurl juga

**Comment by Fucksociety666** [comment]
ok gw tunggu

**Comment by thoriqazzikra** [comment]
> ok gw tunggu

done, check pull ane yg satunya

**Comment by MSI0072** [comment]
> _No description provided._

ngab ini profile buat  liat profile orang lain bisa?

File: Handlemsg.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                    var pfp = pic
+                } 
+                await aruga.sendFileFromUrl(from, pfp, 'pfp.jpg', `*User Profile* ✨️ \n\n➸ *Username: ${namae}*\n\n➸ *User Info: ${status}*\n\n➸ *Admin Group: ${adm}*\n\n`)
+             } else if (quotedMsg) {
+             var qmid = quotedMsgObj.sender.id
+             var pic = await aruga.getProfilePicFromServer(qmid)
+             var namae = quotedMsgObj.sender.name
+             var sts = await aruga.getStatus(qmid)
+             var adm = isGroupAdmins
+             var donate = isAdmin
```
Comment (MSI0072) [inline Handlemsg.js:23]: isAdmin is not defined

Comment (thoriqazzikra) [reply Handlemsg.js:23]: delete yg ini

```var donate = isAdmin```

cari word yg ada kata isAdmin
terus didelete aja

**Pull request closed without merging**